airflow/bin/cli.py

```diff
@@ -363,15 +363,10 @@ def run(args, dag=None):
     task = dag.get_task(task_id=args.task_id)
     ti = TaskInstance(task, args.execution_date)
     ti.refresh_from_db()
-
-    log = logging.getLogger('airflow.task')
-    if args.raw:
-        log = logging.getLogger('airflow.task.raw')
-
-    set_context(log, ti)
+    ti.init_run_context()
 
     hostname = socket.getfqdn()
-    log.info("Running on host %s", hostname)
+    log.info("Running %s on host %s", ti, hostname)
```

Comment:
Think you want `ti.log.info` here.

Reply:
Oh maybe not. I thought `log` wasn't a local variable anymore but tests pass so :)

Reply:
log is actually a local variable as well. I thought it made more sense to log in that facility rather thanb 'hijack' the TI log.